renderer/components/preferences/categories/settings.js

```diff
@@ -80,10 +80,10 @@ class Settings extends React.Component {
           <Switch checked={doNotDisturb} onClick={() => toggleSetting('doNotDisturb')}/>
         </Item>
         <Item
-          title="Loop Kaptures"
+          title="Loop Recordings"
```

Comment:
@skllcrn wanted `Kaptures` to stay in the UI, but let's keep using `recordings` for anything non-UI, like variable names and the settings.